test/Mono.Linker.Tests/TestCasesRunner/AssemblyChecker.cs

```diff
@@ -173,12 +173,13 @@ protected virtual void VerifyTypeDefinitionKept (TypeDefinition original, TypeDe
 				linkedMembers.Remove (f.FullName);
 			}
 
-			foreach (var m in original.Methods) {
-				if (verifiedEventMethods.Contains (m.FullName))
+			foreach (var originalMethod in original.Methods) {
+				if (verifiedEventMethods.Contains (originalMethod.FullName))
 					continue;
-				var msign = m.GetSignature ();
-				VerifyMethod (m, linked?.Methods.FirstOrDefault (l => msign == l.GetSignature ()));
-				linkedMembers.Remove (m.FullName);
+				var methodSignature = originalMethod.GetSignature ();
```

Comment:
Sorry, diff is making this hard to understand -- is this just formatting changes?

Reply:
Yes, just renaming `m` to `originalMethod`